ios/browser/api/ledger/brave_ledger.mm

```diff
@@ -629,12 +628,6 @@ - (void)drainStatusForDrainId:(NSString*)drainId
       });
 }
 
-- (void)transferrableAmount:(void (^)(double amount))completion {
```

Comment:
I see some references to this function in the [brave-ios repo](https://github.com/brave/brave-ios/search?q=transferrableAmount) so it might not be safe to remove it yet.

cc @kylehickinson

Reply:
Good catch — thanks, @zenparsing!
@kylehickinson We'd like to remove `GetTransferableAmount()` from the `Ledger`'s interface. It basically queries the amount of BAT that can be drained when the user verifies their wallet with a custodian. I'm wondering if you still have the use case for it on iOS.

Reply:
Good catch — thanks, @zenparsing!
@kylehickinson We'd like to remove `GetTransferableAmount()` from the `Ledger`'s interface. It basically queries the amount of BAT that can be drained when the user verifies their wallet with a custodian. I'm wondering if you still have the use case for it on iOS.

Reply:
I assume its safe to remove now as I don't think we support draining old wallets into desktop anymore. Once its removed here we'll have to go and remove all the UI + functionality related to it on iOS side.  Maybe we can get confirmation from Jimmy or eV that we're all good to remove?

Reply:
cc: @jsecretan @evq @Miyayes

Reply:
@kylehickinson Just talked to @Miyayes regarding the above: it's safe to remove `transferrableAmount`, and actually the entire draining use case on iOS. Do you want me to create an issue for you?

Reply:
Sounds good!

Reply:
@kylehickinson https://github.com/brave/brave-ios/issues/5632